### DISCUSSION TITLE
Release of ARM architecture version

### DISCUSSION THREAD

**Discussion by Anmolfid1** [claudiodangelis/qr-filetransfer#263] [Q&A] [answered]
Can anyone guide how to make a release from the source code for ARM?

or 

Release ARM version asap.

**Comment by claudiodangelis** [comment, marked as answer]
Hello @Anmolfid1 , I'm sorry I missed your message.
There are already binaries built for ARM, you can check in the releases page.

For the latest version, these are the builds:

https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_arm64.deb
https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_arm64.rpm
https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_arm64.tar
https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_armv7.deb
https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_armv7.rpm
https://github.com/claudiodangelis/qrcp/releases/download/0.10.0/qrcp_0.10.0_linux_armv7.tar
